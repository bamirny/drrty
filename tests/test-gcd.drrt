def int gcd (int x, int y) {
	while (x!=y) {
		if (x>y) {
			x = x - y;
		}else{
			y = y-x;
		}
	return a;
	}	
}

def int main(){
	print(gcd(4,20));
	print(gcd(10,100));

	return 0;

}