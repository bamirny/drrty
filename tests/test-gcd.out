4<br>
10<br>
